app/models/item.rb

```diff
@@ -7,4 +7,8 @@ class Item < ApplicationRecord
   validates :image_name, presence: true
 
   has_many :cart_items, dependent: :destroy
+
+  def stock_dec(quantity)
```

Comment:
例外が発生するメソッドは!をつけたほうがよいですね！

```suggestion
  def stock_dec!(quantity)
```

Reply:
修正しました。